apps/dashr-svm/app.R

```diff
@@ -0,0 +1,514 @@
+library(dashR)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+
+appName <- Sys.getenv("DASH_APP_NAME")
+if (appName != ""){
+  pathPrefix <- sprintf("/%s/", appName)
+
+  Sys.setenv(DASH_ROUTES_PATHNAME_PREFIX = pathPrefix,
+             DASH_REQUESTS_PATHNAME_PREFIX = pathPrefix)
+
+  setwd(sprintf("/app/apps/%s", appName))
+}
+
+source("utils/helperFunctions.R")
+source("utils/reusableComponents.R")
+
+# Read in stored datasets (see data/getData.R to see how data was acquired)
+dataList <- readRDS("data/data.rds")
+
+app <- Dash$new(name = "DashR SVM Explorer")
+
+app$layout(
+  htmlDiv(
+    list(
+      htmlDiv(
+        className = "banner",
+        children = list(
+          htmlDiv(
+            className = "container scalable",
+            children = list(
+              htmlH2(
+                htmlA(
+                  "Support Vector Machine (SVM) Explorer",
+                  href = "https://github.com/plotly/dash-svm",
+                  style = list(
+                    textDecoration = "none",
+                    color = "inherit"
+                  )
+                )
+              ),
+              htmlA(
+                htmlImg(src = "https://user-images.githubusercontent.com/37411533/56820587-55d56e80-681a-11e9-91b6-026e1551b338.png"),
```

Comment:
Let's replace this image with this one, but drop it in your `assets` folder and serve locally:

![logo](https://user-images.githubusercontent.com/9809798/59713053-7a2b4700-91dc-11e9-91b7-ab1b6f7a3c7d.png)